src/api/types/Guide.ts

```diff
@@ -12,9 +12,14 @@ const guideSchema = z.object({
   reviews: z.array(reviewSchema).optional(),
 });
 
-export type Guide = z.infer<typeof guideSchema>;
-
-export type CreateGuideRequest = {
-  title: string;
-  contentHTML: string;
+export type UpdateGuidePayload = {
```

Comment:
delete the update guide payload, we dont need it cause it can be partial type of Guide, as you did here